### PR TITLE
Update README.md to reflect changes for XPU support.

### PR DESCRIPTION
**Pull request by aregm**
Updated to reflect added hardware support and seamless integration with PyTorch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,19 @@
...
 
-# Triton
-
-This is the development repository of Triton, a language and compiler for writing highly efficient custom Deep-Learning primitives. The aim of Triton is to provide an open-source environment to write fast code at higher productivity than CUDA, but also with higher flexibility than other existing DSLs.
+# Intel® XPU Backend for Triton\* 
 
-The foundations of this project are described in the following MAPL2019 publication: [Triton: An Intermediate Language and Compiler for Tiled Neural Network Computations](http://www.eecs.harvard.edu/~htk/publication/2019-mapl-tillet-kung-cox.pdf). Please consider citing this work if you use Triton!
+This is the development repository of Intel® XPU Backend for Triton\*, a new [Triton](https://github.com/triton-lang/triton/) backend for Intel GPUs. Intel® XPU Backend for Triton\* is a a out of tree backend module for [Triton](https://github.com/triton-lang/triton/blob/main/CONTRIBUTING.md) used to provide best-in-class performance and productivity on any Intel GPUs for [PyTorch](https://github.com/triton-lang/triton/blob/main/CONTRIBUTING.md) and standalone usage.
```
Comment (whitneywhtsang) [inline README.md:7]:
```suggestion
This is the development repository of Intel® XPU Backend for Triton\*, a new [Triton](https://github.com/triton-lang/triton/) backend for Intel GPUs. Intel® XPU Backend for Triton\* is a out of tree backend module for [Triton](https://github.com/triton-lang/triton/blob/main/CONTRIBUTING.md) used to provide best-in-class performance and productivity on any Intel GPUs for [PyTorch](https://github.com/triton-lang/triton/blob/main/CONTRIBUTING.md) and standalone usage.
```

File: README.md [hunk 2]
```diff
@@ -176,12 +165,121 @@ For detailed instructions on how to debug Triton's frontend, please refer to thi
...
+Intel® XPU Backend for Triton\* doesn't require any modifications and will work with PyTorch 2.4 release out of the box.
+
+1. Add `import torch` for xpu support.
+2. Put the tensor and models to XPU by calling `to('xpu')`. 
+
+The following examples show modifications for the user code.
+
+### Example 1 : Triton Kernel
+
+This Example is a modified version of [Vector Add](https://triton-lang.org/main/getting-started/tutorials/01-vector-add.html#vector-addition) triton kernel. Please refer to [Vector Add](https://triton-lang.org/main/getting-started/tutorials/01-vector-add.html#vector-addition) for detailed comments and illustration about the code semantics.
```
Comment (whitneywhtsang) [inline README.md:180]:
```suggestion
This example is a modified version of [Vector Add](https://triton-lang.org/main/getting-started/tutorials/01-vector-add.html#vector-addition) triton kernel. Please refer to [Vector Add](https://triton-lang.org/main/getting-started/tutorials/01-vector-add.html#vector-addition) for detailed comments and illustration about the code semantics.
```

**Comment by aregm** [comment]
Closes #929

**Comment by whitneywhtsang** [comment]
Please also fix the pre-commit failure.

**Review by whitneywhtsang — APPROVED** [review]

**Review by pbchekin — APPROVED** [review]

**Pull request merged**